fineract-provider/src/main/java/org/apache/fineract/infrastructure/event/external/producer/ExternalEventProducerImpl.java

```diff
@@ -18,15 +18,16 @@
  */
 package org.apache.fineract.infrastructure.event.external.producer;
 
-import org.apache.fineract.avro.MessageV1;
 import org.apache.fineract.infrastructure.event.external.exception.AcknowledgementTimeoutException;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.stereotype.Service;
 
 @Service
+@ConditionalOnProperty(value = "fineract.events.external.producer.jms.enabled", havingValue = "false")
 public class ExternalEventProducerImpl implements ExternalEventProducer {
```

Comment:
Do we still need this class? If the interface gets instantiated by Spring integration with the proper message gateway, we shouldn't need this anymore, right?

Reply:
This class is required for cases when external event posting is disabled.

Reply:
@ruchiD Is this a dummy impl?

Reply:
yes

Reply:
Please add the "Dummy" prefix to the class name for clarity.